src/ensembl/src/shared/components/radio-group/RadioGroup.tsx

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
+import classNames from 'classnames';
```

Comment:
I get confused with this classNames and the classNames inside props. is that ok? dont they conflict or do you guys not find it confusing, which one is which. Thats why i prefer classNamesMerger.

Reply:
> dont they conflict or do you guys not find it confusing, which one is which

Personally, I don't find them confusing, because the `classNames` that comes from props sits in the `props` object and can easily be accessed with `props.classNames`. So they don't really conflict, and the moment they start conflicting, typescript will yell at you. I think @msmanoj used `classNamesMerger` in some cases where he found having the same name confusing. One thing to keep in mind is that in most of our files we call this imported function classNames, so we kinda got used to the name. Don't really have particularly strong feelings (apart from instead of `classNamesMerger`, which is a noun, I would have used `mergeClassNames` which is a verb and as such better suited to be the name of a function).

Reply:
Thanks for this, i guess for me as i was trying to understand the code and i know for sure specially this file when i was trying to figure out each line, i did get confuse with classNames but as you pointed out and i didnt realise it the props one will always accompany props. and i guess its just a matter getting use with the react code. I am fine with that, thanks for the explanation, it is helpful.